docs/guide/basic-usage.md

```diff
@@ -134,3 +134,19 @@ You can use it by adding it to the `$providers` property of the current command.
     ],
 ]
 ```
+
+
+Yii2 Advanced Template
+----------------------
+ 
+When you want to run faker in the Yii2 advanced template, you need to specify your template and fixtureData path as @tests doesn't exist. For example if you want to setup common fixtures, use the following config in console/config/main.php:
```

Comment:
What does not exist exactly? `@tests`? Why do we need to specify path as `@tests` then?